Product.cs

```diff
@@ -1,7 +1,10 @@
-﻿namespace InventorySystem;
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace InventorySystem;
 
 public class Product
 {
+    [Key]
     public string Name { get ; set; }
```

Comment:
this should give a hint that you should have an int id for your db model instead of a string.